index.js

```diff
@@ -0,0 +1,16 @@
+let names=[]
+function myFunction()
```

Comment:
Please use descriptive variable and function names.

Reply:
Good function name, but going forward can we use [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) except where unavoidable?

So something like `const addNamesToList = () => {} `

Reply:
Alright, sir..I'll refactor the code and submit a PR